Example/DemoOpenTracing/Application/DependencyContainer/ApplicationDependencyContainer.swift

```diff
@@ -24,8 +24,18 @@ class ApplicationDependencyContainer: DependencyContainer {
     let imageDownloader: ImageDownloader = AssetDownloader()
     let demoTracer: WrapTracer
 
-    init() {
-        let sender = JSONSender(endPoint: Const.spanSenderEndPoint)
+    init(jaegerPayload: Bool) {
+        
+        let sender: SpanSender
+        
+        if jaegerPayload {
```

Comment:
`guard` instead of `if-else`?

Reply:
with a guard I would need to create the demoTracer twice, since I need to inject the SpanSender.